src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```diff
@@ -66,7 +66,7 @@
                 setPendingNotificationCount(notificationsInfo);
                 updateModalTitleTotalInfo(notificationsInfo.total);
             })
-            .catch(console.error);
+            .catch();
```

Comment:
Removing error handling is not a solution.
Here should be checked what was the response and do a proper thing to a response.